content/en/docs/concepts/certificate.md

```diff
@@ -54,11 +54,44 @@ it is stored in `ca.crt`.
 
 This format is used as it allows TLS implementations to validate the leaf certificate
 as long as the root CA is already trusted. During the TLS handshake, peers construct
-a full trust chain by checking the issuer of each certificate until they end up at a 
+a full trust chain by checking the issuer of each certificate until they end up at a
 root in their trust store. If the intermediates aren't sent along with the leaf,
 there's no way to know that the issuing CA was signed by the root, and the
 certificate won't be trusted.
 
+
+{{% alert title="Use `ca.crt` with caution" color="warning" %}}
+
+**When configuring a client** to connect to a TLS server with a serving certificate that is signed by a private CA,
+you will need access to the private CA certificate in order to verify the server.
+But if the client is running in a Kubernetes Pod,
+it should not mount the Secret containing the server key and certificate, because:
+
+1. That Secret also contains the private key, which should only be accessible to the server. You should use RBAC to ensure that the Secret containing the serving certificate and key are only accessible to Pods that need it.
+2. When configuring the client you should independently choose and fetch certificate of the CA that you want to trust,
+   download the  `ca.crt` out of band,
+   and store it in a Secret that it is independent of the Secret populated by cert-manager.
+
+This ensures that if the material in Secret containing the server certificates is modified,
+the client will fail to connect to the compromised server.
+
+
+**When configuring a server** to trust to a TLS client with a client certificate that is signed by a private CA,
+you will need access to the private CA certificate in order to verify the client.
+But if the server is running in a Kubernetes Pod,
+it should not mount the Secret containing the client key and certificate, because:
```

Comment:
"To trust to" seemed off but I might be wrong 😅

```suggestion
**When configuring a server** to trust a TLS client with a client certificate that is signed by a private CA,
you will need access to the private CA certificate in order to verify the client.
But if the server is running in a Kubernetes Pod,
it should not mount the Secret containing the client key and certificate, because:
```